test/export/test_export.py

```diff
@@ -213,5 +213,47 @@ def f(x, y):
             export(f, example_inputs, constraints)
 
 
+    @unittest.skipIf(not torchdynamo.is_dynamo_supported(), "dynamo doesn't support")
+    def test_capture_symbolic_tracing(self) -> None:
+        from torch._subclasses import fake_tensor
+        from torch.fx.experimental.symbolic_shapes import ShapeEnv
+        from torch._dynamo.output_graph import config
+
+        class Model(torch.nn.Module):
+            def __init__(self) -> None:
+                super().__init__()
+                self.linear = torch.nn.Linear(2, 2)
+                self.linear2 = torch.nn.Linear(2, 2)
+
+            def forward(self, x):
+                out = self.linear(x)
+                out = self.linear2(out)
+                return out
+
+        # User-instantiated FakeTensorMode
+        fake_mode = fake_tensor.FakeTensorMode(allow_non_fake_inputs=False,
+                                               allow_fallback_kernels=True,
+                                               shape_env=ShapeEnv(
+                                                   allow_scalar_outputs=config.capture_scalar_outputs,
+                                                   allow_dynamic_output_shape_ops=config.capture_dynamic_output_shape_ops,
+                                                   frame_id=0),)
+        # Fakefy input+model before exporting it
+        with fake_mode:
+            x = torch.rand(5, 2, 2)
+            model = Model()
+
+
+        # Export the model with fake inputs and parameters
+        for aten_graph in [True, False]:
+            graph_module, _ = torchdynamo.export(
```

Comment:
Is it required that this call happens outside of `fake_mode` context?

Reply:
Yes, perd Edward's request

Reply:
I see, is it because it's more in line with previous dynamo assumptions? I guess it'll be good to raise with explanation for dynamo if pre-existing fake mode context is detected (not asking to add in this PR), or maybe that's done already?